manifests/i/IrfanSkiljan/IrfanView-plugins/4.58/IrfanSkiljan.IrfanView-plugins.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: IrfanSkiljan.IrfanView-plugins
+PackageVersion: 4.58
+PackageLocale: en-US
+Publisher: Irfan Skiljan
+PublisherUrl: "https://www.irfanview.com/"
+PublisherSupportUrl: https://irfanview-forum.de
+PrivacyUrl: "https://www.irfanview.com/privacy_policy.htm"
+Author: Irfan Skiljan
+PackageName: IrfanView PlugIns
+PackageUrl: "https://www.irfanview.com/plugins.htm"
+License: Proprietary, free for non-commercial use
+LicenseUrl: "https://www.irfanview.com/eula.htm"
+Copyright: Copyright (C) 2021 by Irfan Skiljan, Wiener Neustadt, Austria.
+CopyrightUrl: https://www.irfanview.com/eula.htm
+ShortDescription: Plugins installer for the IrfanView graphics viewer. Adds addition file formats and filters.
+Description: IrfanView is an image viewer, editor, organiser and converter program for Microsoft Windows. The plugins provide additional filters, effects and file format support.
+Moniker: irfanview-plugins
+Tags:
+  - image
+  - image-viewer
+  - photo
+  - picture
+  - graphic-viewer
+  - graphic-editor
+  - Graphics
+  - Plugins
```

```suggestion
  - graphics
  - plugins
```
Oops, forgot about this too.